PHP/ApiClients/CampaignApi.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+include_once 'ClientBase.php';
+include_once 'Http/CurlWrapper.php';
+include_once 'Model/RegisterPageRequest.php';
+include_once 'Model/StoryUpdateRequest.php';
+
+class CampaignApi extends ClientBase
+{		
+	public $Parent;
+	public $curlWrapper;
+	
+	public function __construct($justGivingApi)
+	{
+		$this->Parent		=	$justGivingApi;
+		$this->curlWrapper	= new CurlWrapper();
+	}
+
+	public function Retrieve($charityName, $campaignName)
+	{
+		$locationFormat = $this->Parent->RootDomain . "{apiKey}/v{apiVersion}/campaigns/". $charityName ."/". $campaignName;
+		$url = $this->BuildUrl($locationFormat);
+		$json = $this->curlWrapper->Get($url, $this->BuildAuthenticationValue());
```

Comment:
Here I recommend you to use [GetV2](https://github.com/JustGiving/JustGiving.Api.Sdk/blob/master/PHP/ApiClients/Http/CurlWrapper.php#L20) as it's in [AccountDetails](https://github.com/JustGiving/JustGiving.Api.Sdk/blob/master/PHP/ApiClients/AccountApi.php#L16) because that way we support more information about response.